docs/Behaviour - Client Side.md

```diff
@@ -2,7 +2,17 @@
+Before making an [IS-05][IS-05] connection, an NMOS Controller chooses a Sender and a number of Receivers. The NMOS Controller MAY keep the current configuration of the Sender or change its configuration. In case of changing configuration of an active Sender, deactivating such Sender MAY be required and depends on the Sender itself.
+
+If changing the configuration of the Sender, the NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such processing of them that they can be utilized for building Active Constraints satisfying all (by default) or some (depending on user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. If the processing of the Receiver Capabilities results in an empty `constraint_sets` of Active Constraints, the NMOS Controller SHOULD inform the user that all of the specified Receivers can't be satisfied at the same time. After that the NMOS Controller MUST `PUT /constraints/active` to the Sender and SHOULD make the IS-05 connections of the Receivers if the Constraints have been applied succesfully to the Sender.
+
+If keeping the configuration of the Sender, the NMOS Controller MUST `GET /constraints/supported` and `GET /constraints/active` from the Sender, MUST collect Receiver Capabilities from the Receivers and make such processing of them that they can be utilized for building Active Constraints satisfying all (by default) or some (depending on user preferences) of the Receivers and using Parameter Constraint URNs supported by the Sender. If the Sender supports fewer Parameter Constraint URNs than used in the Receiver Capabilities, the NMOS Controller SHOULD inform the user about it. If the processing of the Receiver Capabilities results in an empty `constraint_sets` of Active Constraints, the NMOS Controller SHOULD inform the user about it.
+Sometimes there can be streams compliant with the current active constraints of the Sender, but not compliant with the resulting `constraint_sets`. In these scenarios the NMOS Controller SHOULD inform the user that the Receivers cannot be accommodated without replacing the Active Constraints.
```

Comment:
I still wonder whether the duplicated steps between the two scenarios isn't helpful.

I've attempted again to extract the commonality to see how that reads:

```suggestion
The NMOS Controller MUST `GET /constraints/supported` from the Sender, MUST collect Receiver Capabilities from the Receivers and process them to build Constraint Sets satisfying all (by default) or some (depending on User preferences) of the Receivers, using Parameter Constraints supported by the Sender.
If the Sender does not support all the Parameter Constraints used in the Receiver Capabilities, the NMOS Controller SHOULD inform the User about it.
If the processing of the Receiver Capabilities results in no Constraint Sets, the NMOS Controller SHOULD inform the User that all of the specified Receivers cannot be satisfied at the same time. 

If changing the configuration of the Sender, the NMOS Controller MUST then `PUT /constraints/active` to the Sender.
If the Constraint Sets have been applied successfully, the NMOS Controller SHOULD then make the IS-05 connections of the Receivers.

If keeping the configuration of the Sender, the NMOS Controller MUST instead `GET /constraints/active` from the Sender and compare these Active Constraints with the Constraint Sets built from the Receiver Capabilities.
Sometimes there can be streams compliant with the current Active Constraints of the Sender, but not compliant with the Constraint Sets built from the Receiver Capabilities.
In these scenarios the NMOS Controller SHOULD inform the User that the Receivers cannot be accommodated without replacing the Active Constraints.
```